artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.User/Infrastructure/Models/UserInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Artificial.Scrum.Master.User.Infrastructure.Models
+{
+    internal class UserInfo
+    {
+        public string UserId { get; set; }
+        public string UserName { get; set; }
```

Comment:
build krzyczy o warningach. Daj nullable albo required na tych stringach. Albo zmień klasę na rekord to samo powinno wtedy ogarnąć

Reply:
rekord nie ogarnal
ale required tak